js/packages/openinference-instrumentation-llama-index/test/llamaIndex.test.ts

```diff
@@ -112,7 +112,7 @@ describe("LlamaIndexInstrumentation", () => {
     expect(spans.length).toBeGreaterThan(0);
 
     // Expect a span for the query engine
-    const queryEngineSpan = spans.find((span) => span.name.includes("Query"));
+    const queryEngineSpan = spans.find((span) => span.name.includes("query"));
     expect(queryEngineSpan).toBeDefined();
   });
```

Comment:
Probably needs some tests here.

Reply:
Added some initial tests. Was taking too much time trying to mock the retrieve method to return scores but just wanted to get initial tests in.